controller/array_action/array_mediator_xiv.py

```diff
@@ -270,10 +270,26 @@ def unmap_volume(self, volume_id, host_name):
             else:
                 raise controller_errors.UnMappingError(vol_name, host_name, ex)
 
-    def get_iscsi_targets_by_iqn(self):
+    def _get_iscsi_targets(self):
+        ip_interfaces = self.client.cmd.ipinterface_list().as_list
```

Comment:
I guess the response of the command is iterable, you don't need as_list here.

Reply:
Done.